script.js

```diff
@@ -1,11 +1,18 @@
 // Create an array called `colors` that contains three colors.
-
+var colors = ["red", "blue","green"]
 // Log the last item in the array to the console.
-
+console.log(colors.length - 1)
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+for (var i = 0; i < 3 i ++)
+console.log("My favourite color is" + [i] )
 // Create an `instructor` object that contains three key-value pairs.
-
+var instructor = {
+  name : "alpha";
+  age : 24;
+  location: "washington dc";
+}
 // Add a `facial-hair` property to `instructor`.
+instructor.facial-hair = "yes";
 
 // Log the value of the `facial-hair` property to the console.
+console.log(instructor[facial-hair:]
```

Comment:
You started to use bracket notation here, but forgot to close the parentheses and added an extra colon at the end.